includes/template-library/sources/remote.php

```diff
@@ -189,6 +189,9 @@ public function get_data( array $args, $context = 'display' ) {
 			return $data;
 		}
 
+		// Set the Request's state as an Elementor upload request, in order to support unfiltered file uploads.
+		Plugin::$instance->uploads_manager->set_elementor_upload_state( true );
```

Comment:
Set `false` after the import (maybe from the upload manager)

Reply:
Done (The upload of the images is not from the upload manager, but it uses the upload manager to check if allow uploading, this is why I revert the `elementor_upload_state` in `remote.php` and not inside the upload manager)